patterns/hidden-sidebar.php

```diff
@@ -8,42 +8,32 @@
 ?>
 <!-- wp:group {"style":{"spacing":{"blockGap":"2.5rem"}},"layout":{"type":"flex","orientation":"vertical","justifyContent":"stretch"}} -->
 <div class="wp-block-group"><!-- wp:group {"layout":{"type":"constrained"}} -->
-<div class="wp-block-group"><!-- wp:image {"id":22,"width":80,"height":80,"scale":"cover","sizeSlug":"full","linkDestination":"none"} -->
-<figure class="wp-block-image size-full is-resized"><img src="<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/team-member-3.webp" alt="" class="wp-image-22" style="object-fit:cover;width:80px;height:80px" width="80" height="80"/></figure>
-<!-- /wp:image -->
+<div class="wp-block-group"><!-- wp:avatar {"size":80,"style":{"border":{"radius":"16px"}}} /-->
 
-<!-- wp:heading {"style":{"typography":{"fontSize":"1.4rem"}}} -->
-<h2 class="wp-block-heading" style="font-size:1.4rem"><?php echo esc_html__( 'About the author', 'twentytwentyfour' ); ?></h2>
-<!-- /wp:heading -->
+<!-- wp:paragraph {"style":{"typography":{"fontSize":"1.4rem"}},"fontFamily":"cardo"} -->
+<p class="has-cardo-font-family" style="font-size:1.4rem"><?php echo esc_html__( 'About the author', 'twentytwentyfour' ); ?></p>
+<!-- /wp:paragraph -->
 
-<!-- wp:post-author-biography {"style":{"typography":{"fontSize":"0.9rem"}}} /-->
+<!-- wp:post-author-biography {"fontSize":"small"} /-->
 </div>
 <!-- /wp:group -->
 
 <!-- wp:separator {"backgroundColor":"base-3"} -->
 <hr class="wp-block-separator has-text-color has-base-3-color has-alpha-channel-opacity has-custom-borders-background-color has-background"/>
 <!-- /wp:separator -->
 
-<!-- wp:heading {"style":{"typography":{"fontSize":"1.6rem"}}} -->
-<h2 class="wp-block-heading" style="font-size:1.6rem"><?php echo esc_html__( 'Latest Posts', 'twentytwentyfour' ); ?></h2>
-<!-- /wp:heading -->
+<!-- wp:paragraph {"style":{"typography":{"fontSize":"1.6rem"}},"fontFamily":"cardo"} -->
+<p class="has-cardo-font-family" style="font-size:1.6rem"><?php echo esc_html__( 'Latest Posts', 'twentytwentyfour' ); ?></p>
```

Comment:
this is coming from the editor, I don't know why it reverted to the old slugs? checking